demo/index.html

```diff
@@ -38,6 +38,7 @@
   selector.addEventListener('change', function() {
     document.documentElement.lang = this.value;
     I18nMsg.lang = this.value;
+    Platform.performMicrotaskCheckpoint();
```

Comment:
That's weird. It exists in polymer-project.org in the console. WebComponents  was aliased to the old Platform a while ago.